src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Generated/ISkillsetsOperations.cs

```diff
@@ -0,0 +1,156 @@
+        /// </param>
+        /// <param name='customHeaders'>
+        /// The headers that will be added to request.
+        /// </param>
+        /// <param name='cancellationToken'>
+        /// The cancellation token.
+        /// </param>
+        /// <exception cref="Microsoft.Rest.Azure.CloudException">
+        /// Thrown when the operation returned an invalid status code
+        /// </exception>
+        /// <exception cref="Microsoft.Rest.SerializationException">
+        /// Thrown when unable to deserialize the response
+        /// </exception>
+        /// <exception cref="Microsoft.Rest.ValidationException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        Task<AzureOperationResponse<Skillset>> GetWithHttpMessagesAsync(string skillsetName, SearchRequestOptions searchRequestOptions = default(SearchRequestOptions), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        /// <summary>
+        /// Creates a new Cognitive Search skillset in an Azure Search service.
+        /// <see href="https://docs.microsoft.com/azure/search/cognitive-search-tutorial-blob" />
+        /// </summary>
+        /// <param name='skillsetName'>
+        /// The name of the skillset to create or update.
+        /// </param>
+        /// <param name='skillset'>
+        /// The skillset containing one or more Cognitive Search skills to
+        /// create or update in an Azure Search service.
+        /// </param>
+        /// <param name='searchRequestOptions'>
+        /// Additional parameters for the operation
+        /// </param>
+        /// <param name='customHeaders'>
+        /// The headers that will be added to request.
+        /// </param>
+        /// <param name='cancellationToken'>
+        /// The cancellation token.
+        /// </param>
+        /// <exception cref="Microsoft.Rest.Azure.CloudException">
+        /// Thrown when the operation returned an invalid status code
+        /// </exception>
+        /// <exception cref="Microsoft.Rest.SerializationException">
+        /// Thrown when unable to deserialize the response
+        /// </exception>
+        /// <exception cref="Microsoft.Rest.ValidationException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        Task<AzureOperationResponse<Skillset>> CreateOrUpdateWithHttpMessagesAsync(string skillsetName, Skillset skillset, SearchRequestOptions searchRequestOptions = default(SearchRequestOptions), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        /// <summary>
+        /// Deletes a Cognitive Search skillset in an Azure Search service.
+        /// <see href="https://docs.microsoft.com/azure/search/cognitive-search-tutorial-blob" />
+        /// </summary>
+        /// <param name='skillsetName'>
+        /// The name of the skillset to delete.
```

Comment:
I think this comment was auto-generated. If you want, we can confirm that and file an issue on AutoRest.

If you want to only review content written by us, it's best to review the Swagger rather than the generated code.

Reply:
I thought this a content-related PR and that these strings would show up in the .NET docs.  If that's not the case, I'm not sure what value I can add in this review

Reply:
They will show up in the .NET docs, once Grok is GA (the .NET API browser no longer supports side-by-side versions of content). 

Content-related feedback is very useful here, it’s just not always possible for our team to act on the feedback ourselves in the case of auto-generated comments.